.github/workflows/cd.yml

```diff
@@ -143,7 +154,7 @@ jobs:
           --yes
 
   release_promote:
-    if: ${{ inputs.tb_deploy}}
+    if: ${{ inputs.tb_deploy && github.event.workflow_dispatch }}
```

Comment:
this is to run this jobs manually

Reply:
OK I added the if in this [PR](https://github.com/tinybirdco/ci/pull/15) not to be triggered but I guess they were executed automatically